tests/test_markov.py

```diff
@@ -1,12 +1,11 @@
 import glob
 import os
 import re
-import shutil
 import unittest
+
```

Comment:
Can you remove the empty line here? `isort` is throwing an error in the CI: https://travis-ci.org/accraze/python-ia-markov/jobs/436443957#L564